Composer/packages/client/src/utils/navigation.ts

```diff
@@ -108,6 +107,18 @@ export function convertDialogDiagnosticToUrl(diagnostic: Diagnostic): string {
   return uri;
 }
 
+export function convertInlineLgPathToUrl(dialogId: string, path: string, templateId: string): string {
+  const tokens = path.split('.');
+  const trigger = tokens[1];
+  const urlToken = tokens.splice(2).join('.');
+  if (templateId.indexOf('bfdactivity') > -1) {
+    return `dialogs/${dialogId}?selected=${trigger}&focused=${trigger}.${urlToken}`;
+  } else if (templateId.indexOf('bfdprompt') > -1) {
+    return `dialogs/${dialogId}?selected=${trigger}&focused=${trigger}.${urlToken}#botAsks`;
```

Comment:
The fragment has some other value:
`export declare enum PromptTab {
    BOT_ASKS = "botAsks",
    USER_INPUT = "userInput",
    OTHER = "other"
}`

Reply:
![dialog](https://user-images.githubusercontent.com/39758135/72447978-3f7ca200-37f1-11ea-8993-ba35132c0a20.gif)